changelog.d/10695.bugfix

```diff
@@ -1 +1 @@
-TODO write a proper changelog dmr.
\ No newline at end of file
+Fixed one case of leaking per-room nicknames and avatar to the user directory, pending further work to tackle other cases.
```

Comment:
```suggestion
Fix leaking per-room nicknames and avatars to the user directory for local users when they update their per-room nickname or avatar.
```

We usually write these as 'Fix' rather than 'Fixed'.

When you come and make another PR, rewrite both changelogs so that they're the same (but also corrected to be what you've actually done in total) — they'll get merged in the final changelog when we do a release.

Equally, if a release goes out before you finish everything else, you still have a sensible log entry that says what you've done.